fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/EraseResourceDAO.java

```diff
@@ -305,9 +305,13 @@ public void deleteFromAllParametersTables(String tablePrefix, long logicalResour
         deleteFromParameterTable(tablePrefix + "_latlng_values", logicalResourceId);
         deleteFromParameterTable(tablePrefix + "_resource_token_refs", logicalResourceId);
         deleteFromParameterTable(tablePrefix + "_quantity_values", logicalResourceId);
+        deleteFromParameterTable(tablePrefix + "_profiles", logicalResourceId);
+        deleteFromParameterTable(tablePrefix + "_tags", logicalResourceId);
```

Comment:
Ouch. Thanks for catching this. This really makes we want to factor these deletes out into their own procedure.